deploy/backplane/srep/10-srep-admins-project.ClusterRole.yml

```diff
@@ -103,3 +110,157 @@ rules:
   - serverstatusrequests
   verbs:
   - '*'
+# configmaps management
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - '*'
+# secrets management
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  resourceNames:
+  - case-management-creds
+  - pull-secret
+  verbs:
+  - '*'
+# limitrange and resourcequota management
+- apiGroups:
+  - ""
+  resources:
+  - limitranges
+  - resourcequotas
+  verbs:
+  - '*'
+# pvc resizing and deletion
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  verbs:
+  - delete
+  - patch
+# serviceaccount creation and deletion
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - create
+  - delete
+# services management (enable_service_loadbalancers.asciidoc)
+- apiGroups:
+  - ""
+  resources:
+  - services
+  verbs:
+  - '*'
+# app scaling
+- apiGroups:
+  - ""
+  resources:
+  - replicationcontrollers/scale
+  verbs:
+  - patch
+- apiGroups:
+  - apps
+  resources:
+  - deployments/scale
+  - replicasets/scale
+  - statefulsets/scale
+  verbs:
+  - patch
+- apiGroups:
+  - apps.openshift.io
+  resources:
+  - deploymentconfigs/scale
+  verbs:
+  - patch
+# replicaset deletion
+- apiGroups:
+  - ""
+  resources:
+  - replicasets
+  verbs:
+  - delete
+  - deletecollection
+# imagestream management
+- apiGroups:
+  - image.openshift.io
+  resources:
+  - imagestreams
+  - imagestreamtags
+  - imagetags
+  verbs:
+  - '*'
+- apiGroups:
+  - image.openshift.io
+  resources:
+  - imagestreamimports
+  - imagestreammappings
+  verbs:
+  - create
+# hive clusterdeployment and machinepool management
```

Comment:
We should create new clusterole/rolebindings for hive specific that only deploy to hive clusters, their namespaces has a "uhc-" prefix which will not works with existing `SubjectPermission` here: https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/srep/20-srep.SubjectPermission.yml#L13